### ISSUE TITLE
Password-Hash  Function Error for Module

### ISSUE THREAD

**Issue by JacobYim** [JacobYim/CSE442-BuyNSell#55]
It has error. 

```
internal/modules/cjs/loader.js:582
    throw err;
    ^

Error: Cannot find module 'password-hash'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:580:15)
    at Function.Module._load (internal/modules/cjs/loader.js:506:25)
    at Module.require (internal/modules/cjs/loader.js:636:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/Users/junghwanyim/CSE442/CSE442-BuyNSell/server.js:5:20)
    at Module._compile (internal/modules/cjs/loader.js:688:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:699:10)
    at Module.load (internal/modules/cjs/loader.js:598:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:537:12)
    at Function.Module._load (internal/modules/cjs/loader.js:529:3)
```

**Comment by JacobYim** [comment]
It is because of the absence of the module

try

```
npm install password-hash
```

It will be resolved

**Comment by JacobYim** [comment]
There is another error

In the branch password-hash,

When run the singin_acceptance_test.py, some test has been failed
```
Junghwan-ui-MacBook-Pro:test junghwanyim$ python3 signin_acceptance_test.py 
Pass test1
Pass test2
Pass test3
Pass test4
Fail test5 : string having ";" case is not blocked.
Fail test6 : string having "." case is not blocked.
Pass test7
Fail test8 : valid signin is failed
Fail test9 : valid signin is failed
Pass test10
```

it is discussed on the issue #56 .

**Comment by JacobYim** [comment]
This issue is solved at #58.
